fs/fserrors/error.go

```diff
@@ -116,6 +116,10 @@ func (err wrappedFatalError) Cause() error {
 	return err.error
 }
 
+func (err wrappedFatalError) Unwrap() error {
```

Comment:
If we aren't using `errors.Is` we don't need the `Unwrap` as we have `Cause` which is what pkg/errors uses.